tests/callbacks/test_tqdm_progress_bar.py

```diff
@@ -568,36 +565,6 @@ def get_metrics(self, trainer: Trainer, model: LightningModule):
     assert "v_num" not in standard_metrics.keys()
 
 
-def test_tqdm_progress_bar_main_bar_resume():
-    """Test that the progress bar can resume its counters based on the Trainer state."""
```

Comment:
I'm not sure about what the intention of this test was. If anybody knows, I can put it back.

Reply:
It was part of the fix #9310 as git blame shows. Resuming the progress bar needed to be taken care of separately which is redundant now that it relies on progress tracking. Given progress tracking is tested.